Xamarin.Forms.Platform.Android/AppCompat/NavigationPageRenderer.cs

```diff
@@ -323,7 +323,7 @@ protected override void OnAttachedToWindow()
 			RegisterToolbar();
 
 			// If there is already stuff on the stack we need to push it
-			PushCurrentPages();
+			PushCurrentPagesAsync();
```

Comment:
I'm surprised we're not getting warnings about not awaiting this.

Reply:
Yea nothing in this files awaits anything

I thought about sprinkling some around but didn't want to modify the timing of anything without proper reasons to

The Tasks here are all just a TaskCompletionSource so none of the tasks need to be awaited to actually trigger anything

Reply:
I mean, at least it's not async void.